src/main/java/org/prebid/server/proto/response/AmpResponse.java

```diff
@@ -1,16 +1,15 @@
 package org.prebid.server.proto.response;
 
+import com.fasterxml.jackson.databind.node.ObjectNode;
 import lombok.AllArgsConstructor;
 import lombok.Value;
 import org.prebid.server.proto.openrtb.ext.response.ExtResponseDebug;
 
-import java.util.Map;
-
 @AllArgsConstructor(staticName = "of")
 @Value
 public class AmpResponse {
 
-    Map<String, String> targeting;
+    ObjectNode targeting;
```

Comment:
Sorry, maybe I wasn't clear enough - I didn't meant replacing `Map<String,Object>` with `JsonNode`, but rather with `Map<String,JsonNode>`.

Reply:
Yep, it seems that I did not understand you but it's because "I can't" :) 

I can perfectly understand the plain ObjectNode/JsonNode approach, but I can not really understand the motivation of using a Map<String, JsonNode>.
In that case, what is the benefit of using Map<String, JsonNode> instead of Map<String, Object> in this scenario ? Nobody is reading that field aside from Jackson when serializing it.

Can you elaborate a little bit more your reasons ?

Reply:
Both `Map<String,Object>` and `Map<String,JsonNode>` would work I assume however latter  would be _consistent_ with how arbitrary JSON objects are defined in all other POJOs in this project and that is something worth seeking. As a side note I must notice that `AmpResponse` _might be_ actually read by `AmpResponsePostProcessor` implementation.